apps/portals/elportal/src/pages/HomePageV2.tsx

```diff
@@ -19,8 +20,27 @@ import {
   topPublicationsSql,
   whatWeDoSql,
 } from '../config/resources'
+import { Typography } from '@mui/material'
+import analyzetheclouds from '@sage-bionetworks/synapse-portal-framework/components/elportal/assets/analyzetheclouds.png'
+import computationaltools from '@sage-bionetworks/synapse-portal-framework/components/elportal/assets/computationaltools.png'
```

Comment:
move these assets to the `elportal` project if they are only used in the elportal

Reply:
do these imports work? I would expect Vite would need you to have the `?url` suffix, but maybe what you have is fine

Reply:
This has been working for me! I assume Vite is configured to handle image assets correctly?